connector/connector_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"go.opentelemetry.io/collector/consumer"
 )
 
-func TestNewConnectorFactory_NoOptions(t *testing.T) {
+func TestNewFactory_NoOptions(t *testing.T) {
```

Comment:
s/TestNewFactory_NoOptions/TestNewFactoryNoOptions

I was told that this is more "go" like :)

Reply:
Moved over here: https://github.com/open-telemetry/opentelemetry-collector/pull/6804